src/data/modifiers.txt

```diff
@@ -2424,58 +2449,53 @@ Item	Mixed Garbanzos and Chickpeas	Maximum HP: [5*(1+skill(Beanweaver))], Maximu
 Item	mole skin notebook	Moxie: +1
 Item	moxie magnet	Moxie Controls MP
 Item	Mr. Balloon	Muscle: +3, Mysticality: +3, Moxie: +3
-# Mr. Haggis: Haggis!
 Item	Mr. Haggis	Damage Aura: 1
+# Mr. Haggis: Haggis!
 Item	mushroom shield	Muscle: +40, Mysticality: -20, Moxie: -20, Combat Rate: +10, Damage Reduction: 6
 # Mylar balloon
-# naughty fortune teller: + Some Stats Per Fight
 Item	naughty fortune teller	Softcore Only
-# Necrotelicomnicon: All Spells Cast Are Spooky
+# naughty fortune teller: + Some Stats Per Fight
 Item	Necrotelicomnicon	Spell Damage: +10
+# Necrotelicomnicon: All Spells Cast Are Spooky
 Item	novelty monorail ticket	Adventures: +3, Rollover Effect: "Favored by Lyle", Rollover Effect Duration: 100
 Item	nozzle of the Phoenix	Mysticality: +15, Mysticality Percent: +25, Hot Spell Damage: +50
 Item	off-hand balloon	Muscle: +1, Mysticality: +1, Moxie: +1
 Item	oil lamp	Sleaze Damage: +5, Hot Damage: +5
-# Ol' Scratch's ash can: All Spells Cast Are Hot
 Item	Ol' Scratch's ash can	Spell Damage Percent: +100, Hot Spell Damage: +30, Class: "Sauceror"
+# Ol' Scratch's ash can: All Spells Cast Are Hot
 Item	Ol' Scratch's stove door	Hot Damage: +20, Damage Reduction: 15, Thorns: 1
 Item	old pizza box	Damage Absorption: +50, Damage Reduction: 6
 Item	old school flying disc	Muscle Percent: +15, Damage Reduction: 24, Familiar Weight: +5
 Item	Operation Patriot Shield	Experience: +3, Maximum HP: +20, Maximum MP: +20, Softcore Only, Variable
-# Operation Patriot Shield: Can be Thrown in Combat
-# Seal Clubber:
-# Operation Patriot Shield	HP Regen Min: 10, HP Regen Max: 12, Weapon Damage: +15, Damage Reduction: 1
-# Turtle Tamer:
-# Operation Patriot Shield	HP Regen Min: 10, HP Regen Max: 12, Familiar Weight: +5
-# Disco Bandit
-# Operation Patriot Shield: Makes you a Crack Shot
-# Operation Patriot Shield	Ranged Damage: +20
-# Accordion Thief
-# Operation Patriot Shield: Lets You Sing More Proudly
```

Comment:
previously, these were class-based (as seen here); they have been reshuffled to be alphabetical.

If the script can be changed to assume that a block of comments should exist in the order they're present, that would be nice.

Reply:
I don't really understand this, it was alphabetical before this change wasn't it? Is this just a really old relic or am I misreading. Tempted to remove.

Reply:
Oh wait I see.

Reply:
The problem is that blocks of comments aren't always right connected. This sucks.

Reply:
you could prefix all of these by "Operation Patriot Shield" to make aligning them easy

otherwise I think we are in
```
# gausie-special-tool-ignore
<stuff>
# no-gausie-special-tool-ignore
```
territory.